Testscripts/Linux/NET-IFUP-IFDOWN.sh

```diff
@@ -89,19 +89,32 @@ done
 LoopCount=4
 TestCount=1
 ChangeInterfaceState "up"
-kill "$(pidof dhclient)"
-dhclient -r && dhclient
+ipAddress=$(ip addr show eth0 | grep "inet\b")
+if [ -z "$ipAddress" ]; then
+    kill "$(pidof dhclient)"
+    dhclient -r && dhclient
+fi
 sleep 15
```

Comment:
if there is no dhclient service starting, can you move sleep statement inside of if statement?